wp-content/themes/core/pcss/utilities/mixins/_typography.pcss

```diff
@@ -25,26 +25,134 @@
  * ------------------------------------------------------------------------- */
 
 @define-mixin body-content {
-	font-size: var(--font-size-content-s);
+	line-height: var(--line-height-base);
+	font-size: var(--font-size-body-s);
 
 	@media (--viewport-medium) {
-		font-size: var(--font-size-content);
+		font-size: var(--font-size-body);
 	}
 }
 
+@define-mixin body-content-large {
```

Comment:
Is this used anywhere? Should it be used for the secondary, larger p implementation?

Reply:
Yes, good catch!